### PR TITLE
Add info about DAWN

### PR DESCRIPTION
**Pull request by llewelld**
Adds general info about how to get access to DAWN.

Slightly refines the instructions for the multli-gpu-accelerate example.

### CODE REVIEW

**Comment by llewelld** [comment]
I've worked through the Multi node accelrate example on DAWN and made a few small changes, also pulling in a simplified version of the `conda env create` script from #1.

Everything worked smoothly on DAWN in my testing today.

**Review by tomaslaz — APPROVED** [review]
LGTM. Tested on the system - everything worked as expected.

**Comment by llewelld** [comment]
Added `#SBATCH --account <>` lines.

File: DAWN/examples/multi_node_accelerate/sbatch_example.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #SBATCH --partition=pvc
```
[the remark this answers is not in the document]
Comment (llewelld) [reply DAWN/examples/multi_node_accelerate/sbatch_example.sh:2]: Thanks for spotting this, it's a really good point. I've added the lines into the files in the latest commit.

File: DAWN/examples/multi_node_accelerate/README.md [hunk 2]
```diff
@@ -1,18 +1,68 @@
...
+
+## Setup
+
+Before starting the training script a Conda environment must be set up containing the required dependencies.
+This can be done by executing the `sbatch_initconda.sh` script using SLURM:
+
+```
+sbatch sbatch_initconda.sh
+```
+
```
[the remark this answers is not in the document]
Comment (llewelld) [reply DAWN/examples/multi_node_accelerate/README.md:35]: Thank you; now added.

File: DAWN/examples/multi_node_accelerate/README.md [hunk 3]
```diff
@@ -1,18 +1,68 @@
...
+7. Line 46: Run a simple test command to check things are working.
+8. Line 49: Run the training script.
+   This line kicks off all the real work that happens in `example.py`.
+
+You should batch execute the script to run the two-node, eight-GPU PyTorch training as described above:
+
+```
+sbatch sbatch_example.sh
+```
+
```
[the remark this answers is not in the document]
Comment (llewelld) [reply DAWN/examples/multi_node_accelerate/README.md:64]: Thanks again for spotting these. Now added.

**Pull request merged**